Automattic/jetpack
# QR Code: remove Jetpack logo from item in Jetpack sidebar

## Proposed changes:

Since we're in the Jetpack sidebar already, we do not need to have an extra Jetpack logo.

<img width="336" alt="Screenshot 2023-04-17 at 19 24 38" src="https://user-images.githubusercontent.com/426388/232563097-460b5bfd-5b7f-47c1-b803-7d97c7a6ffeb.png">

Related: #30138

### Other information:


## Jetpack product discussion

See p9Jlb4-6ZK-p2

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to Posts > Add New
* Write a new post.
* Publish it.
* Once published, open the Jetpack sidebar.
* You should now see a QR code item, with no logo in front of the panel title.
